### ISSUE TITLE
LYRICS DOWNLOAD AS .lrc

### ISSUE THREAD

**Issue by HithaishiDesai** [Sangwan5688/BlackHole#500]
When a song is downloaded also it's lyrics file should get downloaded with .lrc file format (with time synchronisation) so I can play it on my system music app when I am offline

This would really take your app to next level 🔥

THANKS FOR THE BEST MUSIC APP I EVER USED ❤️🙏🏻🇮🇳

**Comment by breaking-loss** [comment]
I agree please make it happen

**Comment by zerobun** [comment]
Yes and if you already have imported synced lyrics on the music file, app should play it as synced lyrics. Cos it seems it jus plays it as plain lyrics instead.
